### DISCUSSION TITLE
节点可用性检查 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1147] [Announcements]
# 节点可用性检查 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/nodes/node-check/

**Comment by tony4gt** [comment]
1、SELinux 以及防火墙如果客户要求不关闭，是否有变通的方法解决？
2、网络联通性中，请给出需要开的最小端口号细节；
3、时间一致性中，是否需要客户有NTP服务器，如果没有的话，是否只能手工调整，满足小于十秒的要求？
